packages/react-native/template/android/utils.gradle

```diff
@@ -0,0 +1,5 @@
+ext.resolveJSModule = { String path ->
```

Comment:
We won't be adding yet another `.gradle` file inside the template (we're actually trying to remove them so this change goes in the wrong direction).